tools/deploy.py

```diff
@@ -43,8 +43,13 @@ def parse_args():
         '--show', action='store_true', help='Show detection outputs')
     parser.add_argument(
         '--dump-info', action='store_true', help='Output information for SDK')
+    parser.add_argument(
```

Comment:
Can we move `--quant` into config file? Tools with too many argument is hard to use.

Reply:
Without `--quant`, how user specify fp32 model ?  
`--quant` + `--quant-image-dir` are minium control option.

Reply:
We have different configs for fp32 and int8. Users can choose what they want or edit the config to meet their demonds.
And you might add more options for quantization such as algorithm or symmetry. Put them all in arguments doesn't sound good.

Reply:
Editing config is not a good use experience.

I have to open doc or source code to understand format, rewrite it cautiously.

Reply:
For example, `use_vulkan=True`. How can I know which backend support vulkan or vulkan int8 now.

Reply:
ok